akka-multi-node-testkit/src/main/mima-filters/2.8.3.backwards.excludes/migrate-to-netty4.backwards.excludes

```diff
@@ -0,0 +1,24 @@
+# Migrate to netty 4
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.remote.testconductor.ClientFSM#Connected.channel")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.remote.testconductor.ClientFSM#Connected.copy")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.remote.testconductor.ClientFSM#Connected.copy$default$1")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.remote.testconductor.ClientFSM#Connected.this")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.remote.testconductor.ClientFSM#Connected.apply")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Connected.unapply")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Data.channel")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Data.copy")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Data.copy$default$1")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Data.this")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Data.apply")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Data.unapply")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.remote.testconductor.Controller#CreateServerFSM.channel")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.remote.testconductor.Controller#CreateServerFSM.copy")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.remote.testconductor.Controller#CreateServerFSM.copy$default$1")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.remote.testconductor.Controller#CreateServerFSM.this")
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.remote.testconductor.Controller#CreateServerFSM.apply")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.Controller#CreateServerFSM.unapply")
+
+# For Scala 3 these are also needed
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.remote.testconductor.ClientFSM#Connected._1")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.remote.testconductor.ClientFSM#Data._1")
+ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.remote.testconductor.Controller#CreateServerFSM._1")
```

Comment:
Are there any changes to user facing API? As far as I can see it's only changing internals (`private[akka]`) ?

Reply:
I ran the mima with scala3 and it report these